### PR TITLE
CDISC Vocabulary introduction

### PR DESCRIPTION
**Pull request by IrynaZherka**

### CODE REVIEW

**Review by TinyRickC137 — CHANGES_REQUESTED** [review]
ICD commits in CDISC pull request

**Review by TinyRickC137 — APPROVED** [review]

**Review by TinyRickC137 — CHANGES_REQUESTED** [review]
as discussed in skype

File: CDISC/manual_work/create_source_tables.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+--Create source table to be filled during 'load_stage' before stage tables created
```
Comment (TinyRickC137) [inline CDISC/manual_work/create_source_tables.sql:1]: Drop file

File: CDISC/load_stage.sql [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+AND b.sab='CDISC' )
+;
+
+-- Update domain for Measurements
+UPDATE concept_stage cs
+SET domain_id = 'Measurement', concept_class_id = 'Procedure'
+WHERE concept_code in
+(
+SELECT s.concept_code
+FROM dev_cdisc.source s
```
Comment (TinyRickC137) [inline CDISC/load_stage.sql:222]: why is there dev_cdisk?

**Review by TinyRickC137 — APPROVED** [review]

**Review by AlaikseiKatyshou — APPROVED** [review]

File: CDISC/load_stage.sql (hunk 2 above)
Comment (vladkorsik) [reply CDISC/load_stage.sql]: was preserved for one LK only

**Pull request merged**